### PR TITLE
Use new `fs` methods for file copying

### LINKED ISSUE #7: Use the new `fs` method for doing the file copying
http://yeoman.io/blog/release-0.18.html

Help welcome :)

### PR DESCRIPTION
**Pull request by kevva**
Fixes #7.

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -36,29 +36,31 @@ module.exports = yeoman.generators.Base.extend({
...
+			this.fs.copy(this.templatePath('.travis.yml'), this.destinationPath('.travis.yml'));
+			this.fs.copyTpl(this.templatePath('index.js'), this.destinationPath('index.js'), tmpl);
+			this.fs.copyTpl(this.templatePath('license'), this.destinationPath('license'), tmpl);
 			// needed so npm doesn't try to use it and fail
-			this.template('_package.json', 'package.json');
-			this.template('_readme.md', 'readme.md');
-			this.template('test.js');
+			this.fs.copyTpl(this.templatePath('_package.json'), this.destinationPath('package.json'), tmpl);
+			this.fs.copyTpl(this.templatePath('readme.md'), this.destinationPath('readme.md'), tmpl);
+			this.fs.copyTpl(this.templatePath('test.js'), this.destinationPath('test.js'), tmpl);
```
Comment (sindresorhus) [inline app/index.js]: https://github.com/sindresorhus/generator-gulp-plugin-boilerplate/commit/d7503c081f025e6ce2f8665638d353e444ffc6e6#commitcomment-11796951

**Comment by sindresorhus** [comment]
ping @kevva :)

**Comment by kevva** [comment]
Sorry, forgot about this :). I think most stuff should be fixed now.

**Comment by kevva** [comment]
ping @sindresorhus

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**